eng/Analyzers.props

```diff
@@ -4,4 +4,16 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="$(MicrosoftCodeAnalysisCSharpCodeStyleVersion)" PrivateAssets="all" />
     <!-- <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.205" PrivateAssets="all" /> -->
   </ItemGroup>
+
+  <Import Project="$(BootstrapBuildPath)/Microsoft.NET.ILLink.Analyzers.props" Condition="'$(BootstrapBuildPath)' != ''" />
+
+  <!-- Don't enable for Cecil, as they can't be suppressed -->
+  <PropertyGroup Condition="'$(BootstrapBuildPath)' != '' and '$(MSBuildProjectName)' != 'Mono.Cecil.Pdb'">
```

Comment:
Why just `Mono.Cecil.Pdb` and not the other cecil projects (mainly the `Mono.Cecil` itself)?

Reply:
Mono.Cecil.PDB has a warning (COM) that I need to file a bug for.

We also can't disable it because it sets `<NoWarn>` without using `$(NoWarn)` inside, so it overrides the higher warnings. None of the other projects do, so we can temporarily disable warnings for them.

Reply:
We could push a change to the project to correctly pick up `NoWarn` from the outside.
Once we have XML support it should be possible to get rid of the warning that way.